tests/Unit/SDK/SdkTest.php

```diff
@@ -76,7 +77,8 @@ public function test_getters(): void
         $propagator = $this->createMock(TextMapPropagatorInterface::class);
         $meterProvider = $this->createMock(MeterProviderInterface::class);
         $tracerProvider = $this->createMock(TracerProviderInterface::class);
-        $sdk = new Sdk($tracerProvider, $meterProvider, $propagator);
+        $loggerProvider = $this->createMock(LoggerProviderInterface::class);
+        $sdk = new Sdk($tracerProvider, $meterProvider, $loggerProvider, $propagator);
         $this->assertSame($propagator, $sdk->getPropagator());
         $this->assertSame($meterProvider, $sdk->getMeterProvider());
         $this->assertSame($tracerProvider, $sdk->getTracerProvider());
```

Comment:
**question**: Do we need to have a:
`$this->assertSame($tracerProvider, $sdk->getLoggerProvider());`